{"failedTests":[{"testName":"parallel/test-without-async-context-frame","timestamp":1737733829932,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2019-COMPILED_BY-vs2022 #32250","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2019-COMPILED_BY-vs2022/32250/","buildNumber":32250,"builtOn":"test-rackspace-win2019_vs2019-x64-3","nodeVersion":"24.0.0","commitHash":"ef9cb6d83fc7362e224bac80a348dca356076b37","callStack":[{"upstreamBuild":68070,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":77279,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":64712,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"56712"}],"tap":{"ok":false,"name":"parallel/test-without-async-context-frame","id":994,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ without AsyncContextFrame\n  ✖ async-hooks\\test-async-local-storage-args.js (4350.2984ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:981:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1360:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:297:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-async-await.js (2272.1281ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (1137.6934ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (1159.3803ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (1183.7855ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (1218.5532ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (1233.3392ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (1234.5883ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (1321.9572ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (1285.8492ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (1175.0402ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (1109.8751ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (1620.3631ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (1306.3668ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (1270.1462ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (1153.4268ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (1552.1838ms)\n  ✔ parallel\\test-async-local-storage-bind.js (1415.0611ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (1289.1126ms)\n  ✖ parallel\\test-async-local-storage-deep-stack.js (1153.5244ms)\n    AssertionError [ERR_ASSERTION]: Test parallel\\test-async-local-storage-deep-stack.js failed with exit code 3221225477\n    \n    3221225477 !== 0\n    \n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:981:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:678:7) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (1385.4761ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (1662.3219ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (1250.0535ms)\n✖ without AsyncContextFrame (33758.1469ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 33771.8427\n\n✖ failing tests:\n\ntest at test\\parallel\\test-without-async-context-frame.mjs:49:5\n✖ async-hooks\\test-async-local-storage-args.js (4350.2984ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-args.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:981:9)\n      at async Promise.all (index 0)\n      at async Suite.run (node:internal/test_runner/test:1360:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:297:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-without-async-context-frame.mjs:49:5\n✖ parallel\\test-async-local-storage-deep-stack.js (1153.5244ms)\n  AssertionError [ERR_ASSERTION]: Test parallel\\test-async-local-storage-deep-stack.js failed with exit code 3221225477\n  \n  3221225477 !== 0\n  \n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-without-async-context-frame.mjs:59:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:981:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:678:7) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":34324.283,"fullname":"parallel/test-without-async-context-frame","closingTestPoint":false}},{"testName":"parallel/test-fs-cp","timestamp":1737733829932,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win11-COMPILED_BY-vs2022 #32250","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022/32250/","buildNumber":32250,"builtOn":"test-azure_msft-win11_vs2022-x64-4","nodeVersion":"24.0.0","commitHash":"ef9cb6d83fc7362e224bac80a348dca356076b37","callStack":[{"upstreamBuild":68070,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":77279,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":64712,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"56712"}],"tap":{"ok":false,"name":"parallel/test-fs-cp","id":304,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"node:assert:128\n  throw new AssertionError(obj);\n  ^\n\nAssertionError [ERR_ASSERTION]: Expected values to be strictly equal:\n+ actual - expected\n\n+ 'ERR_FS_EISDIR'\n- 'ERR_FS_CP_EINVAL'\n          ^\n\n    at file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-fs-cp.mjs:687:12\n    at c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\index.js:456:15\n    at node:fs:190:23\n    at callbackifyOnRejected (node:util:230:10)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {\n  generatedMessage: true,\n  code: 'ERR_ASSERTION',\n  actual: 'ERR_FS_EISDIR',\n  expected: 'ERR_FS_CP_EINVAL',\n  operator: 'strictEqual'\n}\n\nNode.js v24.0.0-pre"},"time":546.011,"fullname":"parallel/test-fs-cp","closingTestPoint":false}}]}